.github/workflows/lint.yml

```diff
@@ -0,0 +1,45 @@
+name: Linters
+on:
+  push:
+    branches-ignore:
+    - main
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  markdownlint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Perform lint wirh markdownlint-cli
+      uses: nosborn/github-action-markdown-cli@v3.3.0
+      with:
+        files: .
+        config_file: .markdownlint.yaml
+        dot: false
+
+  format-hcl:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Check linting of Terraform modules
+      uses: devops-infra/action-tflint@v0.3
+      with:
+        dir_filter: terraform/modules/eks,terraform/modules/iam,terraform/modules/vpc
+
+  tfsec:
+    name: tfsec
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Terraform security scan
+      uses: triat/terraform-security-scan@v3.0.0
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
```

Comment:
change to 
```
GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
No. That's token from the [github context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context).